doc/pages/administrator/066-migrate_to_astartedefaultingress.md

```diff
@@ -0,0 +1,135 @@
+# Migrating to the Astarte Default Ingress
+
+If your Astarte deployment is still being served by the deprecated AstarteVoyagerIngress, it is
+highly advised to move to the new managed ingress, namely, the AstarteDefaultIngress.
+
+The current page focuses on describing the procedure for migrating your ingress with ease, simply
+employing `astartectl`.
+
+Please, make sure you have read the entirety of this page and to understand all the concepts and the
+implications before performing the actual migration.
+
+## Prerequisites and preliminary checks
+
+Before starting with the actual migration procedure, some preliminary activities are required:
+
+- ensure that the version of the Astarte operator in your cluster is at least `>= v1.0.1` and
+  stable. If this requirement is not fulfilled, please refer to the [Upgrade
+  Procedures](000-upgrade_index.html) section;
+
+- the [`ingress-nginx`](https://kubernetes.github.io/ingress-nginx/) ingress controller must be
+  deployed in your cluster (see [this section](020-prerequisites.html#nginx) for the details).
+  Be sure of taking note of the `ingress-class` of the controller which is meant to handle your
+  Astarte ingress. This information will come handy during the migration itself;
+
+- make sure the TLS secrets used to secure the communications to and from Astarte are deployed. For
+  the details, please refer to the [Handling Astarte certificates](050-handling_certificates.html)
+  page;
+
+- ensure that [`astartectl`](https://github.com/astarte-platform/astartectl) is installed on your
+  machine and its version is at least `>= v1.0.0`.
+
+## Performing the migration
+
+Performing the actual ingress migration is as simple as executing an `astartectl` command:
+
+```bash
+$ astartectl cluster instances migrate replace-voyager \
+  --namespace <astarte-namespace> \
+  --ingress-name <astartevoyageringress-name> \
+  --operator-name <astarte-operator-name> \
+  --operator-namespace <astarte-operator-namespace>
```

Comment:
We usually display commands assuming default values. Should optional flags be discussed below, instead?

Reply:
If flag descriptions will be added below, it would be nice to put the `kubectl` commands to get those names, e.g. `kubectl get avi -n <astarte-namespace> for the AVI name